recipe/meta.yaml

```diff
@@ -0,0 +1,179 @@
+# occurrences of the load() function
+{% set ignore_modules = ignore_modules + " --ignore=test/torchaudio_unittest/datasets/tedlium_test.py" %}
+# skipping this test because of URL errors on osx (these are passing locally though)
+{% set ignore_modules = ignore_modules + " --ignore=test/integration_tests/loudness_compliance_test.py" %}  # [osx]
+# stream writer tests fail on windows because another process locks the writing on the output directory (common problem on win)
+{% set ignore_modules = ignore_modules + " --ignore=test/torchaudio_unittest/io/stream_writer_test.py" %}  # [win]
+
+test:
+  source_files:
+  {% if run_all_unittests %}
+    - test/torchaudio_unittest
+  {% else %}
+    # choosing a subset of tests to prevent resource exhaustion and still run a reasonable amount of unit tests
+    - test/torchaudio_unittest/assets
+    - test/torchaudio_unittest/backend
+    - test/torchaudio_unittest/common_utils
+    - test/torchaudio_unittest/compliance
+    - test/torchaudio_unittest/datasets/__init__.py
+    - test/torchaudio_unittest/datasets/tedlium_test.py
+    - test/torchaudio_unittest/example
+    - test/torchaudio_unittest/functional/__init__.py
+    - test/torchaudio_unittest/functional/functional_cpu_test.py
+    - test/torchaudio_unittest/functional/autograd_impl.py
+    - test/torchaudio_unittest/functional/torchscript_consistency_impl.py
+    - test/torchaudio_unittest/io
+    - test/torchaudio_unittest/models/conformer
+    - test/torchaudio_unittest/models/decoder
+    - test/torchaudio_unittest/models/rnnt
+    - test/torchaudio_unittest/models/rnnt_decoder
+    - test/torchaudio_unittest/transforms/__init__.py
+    - test/torchaudio_unittest/transforms/torchscript_consistency_impl.py
+    - test/torchaudio_unittest/transforms/transforms_test.py
+    - test/torchaudio_unittest/transforms/transforms_cpu_test.py
+    - test/torchaudio_unittest/transforms/transforms_test_impl.py
+    - test/torchaudio_unittest/transforms/autograd_test_impl.py
+    - test/torchaudio_unittest/transforms/autograd_cpu_test.py
+    - test/torchaudio_unittest/utils
+    - test/torchaudio_unittest/__init__.py
+  {% endif %}
+    - test/smoke_test
+    - test/integration_tests
+    - examples
+  imports:
+    - torchaudio
+  commands:
+    - pip check
+    - python test/smoke_test/smoke_test.py
+    - pytest -v {{ ignore_modules }} test/integration_tests
+    # dataset provides the dataset for the hubert example test. It's not a runtime requirement.
+    - export PYTHONPATH=$PWD/examples/hubert:$PYTHONPATH  # [not win]
+    - set PYTHONPATH=%cd%\examples\hubert;%PYTHONPATH%  # [win]
+    # We don't provide kaldi-io in our distro, so skip tests that test functionality that requires it
+    # also, skipping on osx-64 because of resource exhaustion
+    - pytest -v {{ ignore_modules }} -k "not kaldi" test/torchaudio_unittest  # [not (osx and x86_64)]
```

Comment:
One test failed on `win64`:
`FAILED test/torchaudio_unittest/io/effector_test.py::EffectorTest::test_formats_5_mp3`

And 14 tests failed on `s390x`

Reply:
I just investigated them. Failed test on `win-64` is because of `ffmpeg` not supporting Lame-mp3 on win. `s390x` was skipped because of documented issues with `save()`.